universe/UniverseObject.h

```diff
@@ -29,6 +29,10 @@ FO_COMMON_API extern const int INVALID_OBJECT_ID;
 // The ID number assigned to temporary universe objects
 FO_COMMON_API extern const int TEMPORARY_OBJECT_ID;
 
+// The ID number assigned instead of the system which represents the whole imperium
+// Used for industrial stockpile
+FO_COMMON_API extern const int IMPERIAL_OBJECT_ID;
```

Comment:
This is a fake id (set to a constant value) and has no connection to empire id (which i would dynamically need to set/find out). 
This gets is used to compare against planets.
One question - are the ids of objects globally unique or just per type (so could be there an empire id 1 == and a planet id == 1 or not)?